pkg/usermanagement/client.go

```diff
@@ -588,6 +588,50 @@ type ListIdentitiesOpts struct {
 	ID string `json:"id"`
 }
 
+type Session struct {
+	// The Object will always be "session".
+	Object string `json:"object"`
+	// The unique identifier for the Session.
+	ID string `json:"id"`
+	// The unique identifier for the User.
+	UserID string `json:"user_id"`
+	// The unique identifier for the Organization.
+	OrganizationID string `json:"organization_id"`
+	// The status of the Session.
+	Status string `json:"status"`
+	// The authentication method used to create the Session.
+	AuthMethod string `json:"auth_method"`
+	// The IP address from which the Session was created.
+	IPAddress string `json:"ip_address"`
+	// The user agent from which the Session was created.
+	UserAgent string `json:"user_agent"`
+	// The time at which the Session expires.
+	ExpiresAt time.Time `json:"expires_at"`
+	// The time at which the Session ended, if applicable.
+	EndedAt *time.Time `json:"ended_at"`
+	// The time at which the Session was created.
+	CreatedAt time.Time `json:"created_at"`
+	// The time at which the Session was last updated.
+	UpdatedAt time.Time `json:"updated_at"`
+}
+
+type ListSessionsOpts struct {
+	UserID string `url:"user_id,omitempty"`
```

Comment:
`UserID` goes in the URL path for this endpoint, so we'll either want to separate it from opts, or make it unexported by renaming it `userID`.